src/SFML/Window/JoystickImpl.hpp

```diff
@@ -47,11 +47,11 @@ struct JoystickCaps
     JoystickCaps()
     {
         buttonCount = 0;
-        std::fill(axes, axes + Joystick::AxisCount, false);
+        std::fill(std::begin(axes), std::end(axes), false);
```

Comment:
If we use std::array, we won't need this

Reply:
Won't fix as discussed live